libmscore/fret.cpp

```diff
@@ -39,6 +39,7 @@ static const ElementStyle fretStyle {
       { Sid::fretStrings,                        Pid::FRET_STRINGS            },
       { Sid::fretFrets,                          Pid::FRET_FRETS              },
       { Sid::fretNut,                            Pid::FRET_NUT                },
+      { Sid::fretMinDistance,                    Pid::MIN_DISTANCE            },
```

Comment:
Thanks for adding this!  I guess a merge conflict is still likely just because of the proximity of our changes - mine is #5027 - but if yours get merged first, I can actually back out that portion of mine.